README.md

```diff
@@ -1,35 +1,35 @@
 # Kubeflow + MLflow on Juju with Microk8s
 
-![screenshot](demo.png "Screenshot showing kubeflow notebook publishing to mlflow")
+## Get Started
```

Comment:
Should we offer both a with and without kubeflow option here?  Without kubeflow works too and is pretty simple, something like:
```
juju deploy mlflow-server
juju deploy charmed-osm-mariadb-k8s mlflow-db
juju deploy minio
juju relate minio mlflow-server
juju relate mlflow-db mlflow-server
```

At minimum, that'll be helpful for us debugging/developing